floodviz/templates/hydrograph.html

```diff
@@ -3,32 +3,84 @@
 
 <head>
 	<meta charset="UTF-8">
-	<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='bower_components/bootstrap/dist/css/bootstrap.css') }}" />
+	<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
+	<link rel="stylesheet" type="text/css"
+		  href="{{ url_for('static', filename='bower_components/bootstrap/dist/css/bootstrap.css') }}"/>
 	<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/style.css') }}"/>
-	<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/hydrograph.css') }}"/>
+	<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/map.css') }}" />
+	<link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='css/hydrograph.css') }}" />
 	<title>Flood Viz</title>
 
+	<script type="text/javascript">
+		(function (i, s, o, g, r, a, m) {
+			i['GoogleAnalyticsObject'] = r;
+			i[r] = i[r] || function () {
+					(i[r].q = i[r].q || []).push(arguments)
+				}, i[r].l = 1 * new Date();
+			a = s.createElement(o),
+				m = s.getElementsByTagName(o)[0];
+			a.async = 1;
+			a.src = g;
+			m.parentNode.insertBefore(a, m)
+		})(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');
+
+		ga('create', 'UA-103831764-1', 'auto');
+		ga('send', 'pageview');
+
+		// Global Namespace for project - FV
+		var FV = FV || {};
+
+		// Wrapper function for Google Analytics explicit send
+		FV.ga_send_event = function(category, action, label) {
+			ga('send', {
+				hitType: 'event',
+				eventCategory: category,
+				eventAction: action,
+				eventLabel: label
+			});
+		}
+
+	</script>
+
 </head>
 
 <body>
 
-	<div class="main-content container-fluid">
-		<div id="hydrograph"><svg></svg></div>
-		<div id="map"></div>
+	<div class="main-content container">
+		<div class="row">
+			<div class="col-sm-12 col-centered">
+				<div id="hydrograph"></div>
+			</div>
+		</div>
 	</div>
 
+	<div id="map"><style>#map {height: 0; width: 0;} #maptip {opacity: 0;}</style></div>
```

Comment:
Why is the map in here? Please remove. Also I suspect you don't need the div for the class "row" and can probably get rid of col-sm-12